src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -408,4 +408,44 @@ else if (sibling.getType() == TokenTypes.SINGLE_LINE_COMMENT) {
                 || BlockCommentPosition.isOnPackage(blockComment));
     }
 
+    /**
+     * Gets the content of inline custom tag.
+     *
+     * @param javadocInlineTag javadoc inline tag node.
+     * @return String consisting of the content of inline custom tag.
+     */
+    public static String getContentOfInlineCustomTag(DetailNode javadocInlineTag) {
+        final DetailNode[] childrenOfJavadocInlineTag = javadocInlineTag.getChildren();
+        DetailNode nodeToBeProcessed = childrenOfJavadocInlineTag[2];
+        final DetailNode endingNode =
+                childrenOfJavadocInlineTag[childrenOfJavadocInlineTag.length - 1];
+        final StringBuilder customTagContent = new StringBuilder(520);
```

Comment:
one minor:
520 seems like a magic number and I do not get why it is like this. 
Such number should be either justified, or just use parameterless ctor with default value.

Reply:
I used parameterless initially, TeamCity complained about it so had to give it a number.
> 520 seems like a magic number

I feel the same but I had some input https://stackoverflow.com/questions/13360229/most-efficient-initial-capacity-size-for-stringbuilder (didn't test it at my end)

Reply:
I see that in other places in code mostly 256 or 1024 used. Lets make it 256

Reply:
done